web/src/flow/stages/identification/IdentificationStage.ts

```diff
@@ -44,23 +45,39 @@ export class IdentificationStage extends BaseStage<
 > {
     form?: HTMLFormElement;
 
+    @query("ak-stage-identification-password")
+    passwordField!: HTMLInputElement;
+
+    @query("ak-stage-identification-toggle-password-visibility")
+    visibilityToggle!: HTMLButtonElement;
+
     static get styles(): CSSResult[] {
-        return [PFBase, PFAlert, PFLogin, PFForm, PFFormControl, PFTitle, PFButton].concat(css`
+        return [
+            PFBase,
+            PFAlert,
+            PFInputGroup,
+            PFLogin,
+            PFForm,
+            PFFormControl,
+            PFTitle,
+            PFButton,
             /* login page's icons */
```

Comment:
Although this is in response to my adding PFInputGroup, I moved this comment _out_ of the CSS; there's no reason to ship it with what the user sees.